TelCoColorCoderVars.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#ifndef TELCOCOLORCODERVARS_H
+#define TELCOCOLORCODERVARS_H
+namespace TelCoColorCoder {
+    enum MajorColor { WHITE, RED, BLACK, YELLOW, VIOLET };
+    enum MinorColor { BLUE, ORANGE, GREEN, BROWN, SLATE };
+
+    extern const char* MajorColorNames[];

```

Comment:
There's a constraint here - the number of strings in this array need to be same as the number of enums.
What if another developer adds an enum and forgets to add the name in this array?
Is it possible to catch such errors early? At compile time? In unit-tests?